runtime/lua/vim/lsp/inlay_hint.lua

```diff
@@ -94,6 +94,10 @@ function M.on_refresh(err, _, ctx)
   for _, bufnr in ipairs(vim.lsp.get_buffers_by_client_id(ctx.client_id)) do
     for _, winid in ipairs(api.nvim_list_wins()) do
       if api.nvim_win_get_buf(winid) == bufnr then
+        local bufstate = bufstates[bufnr]
+        if bufstate then
```

Comment:
unnecessary local declaration?
```suggestion
        if bufstates[bufnr] then
```